apps/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -587,9 +587,14 @@ export class WorkspaceInstallManager extends BaseInstallManager {
       args.push('--recursive');
       args.push('--link-workspace-packages', 'false');
 
-      // "<package>..." selects the specified package and all direct and indirect dependencies
-      for (const toProject of this.options.toProjects) {
-        args.push('--filter', `${toProject.packageName}...`);
+      const filteredProjects: string[] = this.options.toProjects
```

Comment:
For ease of reading I would suggest just doing this as a separate for loop

Reply:
See changes, is this what you meant?

Reply:
Done!